deepchecks/core/serialization/check_result/wandb.py

```diff
@@ -32,12 +32,11 @@
         'To get it, run "pip install wandb".'
     ) from e
 
-try:
-    if t.TYPE_CHECKING:
-        from wandb.sdk.data_types.base_types.wb_value import \
-            WBValue  # pylint: disable=unused-import
-except ImportError:
-    pass
+
+if t.TYPE_CHECKING:
+    from wandb.sdk.data_types.base_types.wb_value import \
+        WBValue  # pylint: disable=unused-import
```

Comment:
you should keep the try catch because it was only added to the newest wandb version and it is only for type checking and we shouldn't make users update it

Reply:
from python docs

>typing.TYPE_CHECKING - A special constant that is assumed to be True by 3rd party static type checkers. It is False at runtime.

I am not sure if that try-except is needed here/ in general it will never raise an error, and static type checker probably will mark it as unresolved import/

Reply:
oh ok